spec/menu_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'menu'
+
+describe Menu do 
+  let(:thai_menu) {Menu.new}
+  let(:green_curry_details) {{:dish => 'green curry', :price => 5.99}}
+  let(:dish) {'green curry'}
+  let(:quantity) {2}
+  
+  context ' initialize' do
+
+    it 'on #display will show list of menu items' do 
+        expect(thai_menu.list).to include(green_curry_details)
```

Comment:
Is testing an expected inclusion as rigourous as testing a full equality exepctatiom?